packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTree.test.tsx

```diff
@@ -175,34 +175,35 @@ describe("ModelsTree", () => {
         const node = createModelNode();
         setupDataProvider([node]);
 
-        visibilityHandlerMock.setup(async (x) => x.getVisibilityStatus(moq.It.isAny())).returns(async () => ({ state: "hidden", isDisabled: false })).verifiable(moq.Times.exactly(2));
-        const result = render(<ModelsTree {...sizeProps} iModel={imodelMock.object} modelsVisibilityHandler={visibilityHandlerMock.object} activeView={mockViewport().object} />);
+        visibilityHandlerMock.getVisibilityStatus = async () => ({ state: "hidden", isDisabled: false });
+        const getVisibilityStatusSpy = sinon.spy(visibilityHandlerMock, "getVisibilityStatus");
```

Comment:
```suggestion
        const getVisibilityStatusSpy = sinon.stub().resolves({ state: "hidden", isDisabled: false });
        visibilityHandlerMock.getVisibilityStatus = getVisibilityStatusSpy;
```

This suggestion also applies to some other tests.